easybuild/tools/hooks.py

```diff
@@ -120,10 +121,7 @@ def load_hooks(hooks_path):
 
 def verify_hooks(hooks):
     """Check whether list of obtained hooks only includes known hooks."""
```

Comment:
here hooks is also a dict, right?
and also in easybuild/framework/easyblock.py line 147?